skinStyles/extensions/StructuredNavigation/ext.structuredNav.NavigationView.less

```diff
@@ -0,0 +1,49 @@
+/*
+ * Citizen
+ *
+ * SkinStyles for StructuredNavigation
+ * Module: ext.structuredNav.NavigationView.styles
+ * Version: REL1_39 55e2ec0
+ *
+ * Date: 2024-09-06
+*/
+
+/* ext.structuredNav.NavigationView.less */
+.mw-structurednav-navigation-container {
+	background-color: var(--color-surface-2);
```

Comment:
Use `var( --color-surface-0 )` instead. It is the equivalent of  `@background-color-base` in the original style.